tutorial_sarscov2.md

```diff
@@ -0,0 +1,231 @@
+
+```bash
+snakemake -s vpipe.snake --use-conda -p --cores 2
+```
+
+
+## Larger dataset
+
+Log onto [Euler](https://scicomp.ethz.ch/wiki/Euler) with ETH account.
+
+```bash
+ssh login.euler.ethz.ch
+```
+
+There is a much larger test data that you can use:
+
+```bash
+ls /cluster/work/bewi/members/mpirkl/Sars-Cov-2_data/
+ls /cluster/work/bewi/members/mpirkl/Sars-Cov-2_data/PRJNA6*
+ls /cluster/work/bewi/members/mpirkl/Sars-Cov-2_data/PRJNA6*/sample*/
+```
+
+## Cluster deployment
+
+It is possible to ask snakemake [submit jobs on a cluster](https://snakemake.readthedocs.io/en/stable/executing/cluster-cloud.html#cluster-execution), such as [LSF system used on Euler](https://scicomp.ethz.ch/wiki/Using_the_batch_system)
+
+
+
+
+```bash
+# Linux:
+wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
+sh Miniconda3-latest-Linux-x86_64.sh -b -p $SCRATCH/miniconda3
+
+# Mac OS X:
+curl -O https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-x86_64.sh
+sh Miniconda3-latest-MacOSX-x86_64.sh -b -p $SCRATCH/miniconda3
+
+$SCRATCH/miniconda3/bin/conda install -c bioconda snakemake-minimal
+```
+> (We can install the *-minimal* version of snakemake: we will probably not run any GUI functionality)
+
+You might get versions conflict at this point, specially if you are re-using your older miniconda version which could be outdated. Consider updating it:
+
+```bash
+$SCRATCH/miniconda3/bin/conda update conda
+```
+
+And/or if you have a recent enough `conda` executable, you can also consider placing your requirement in a different prefix, like we did in the workstation/laptop section of the tutorial:
+
+```bash
+$SCRATCH/miniconda3/bin/conda create -p $SCRATCH/V-pipe_conda -c bioconda conda snakemake
+$SCRATCH/V-pipe_conda/bin/snakemake --version
+```
+
+Let's fetch V-pipe:
+
+```bash
+cd $SCRATCH
+git clone -b sars-cov2 https://github.com/cbg-ethz/V-pipe.git
+cd V-pipe
+```
+
+> **Tips:** There are [snakemake parameters for conda](https://snakemake.readthedocs.io/en/stable/executing/cli.html#CONDA)
+that can help manage:
+> - using `--create-envs-only`, it's possible to only download the dependencies, but not run the pipeline itself
+> - using `--conda-prefix {DIR}`  will store the conda environments of dependencies in that directory (thus possible to share re-use between multiple instances of V-pipe)
+
+```bash
+. $SCRATCH/miniconda3/bin/activate
+
+# download everything in advance
+$SCRATCH/miniconda3/bin/snakemake -s vpipe.snake --use-conda --conda-prefix $SCRATCH/snake-envs --cores all --create-envs-only
+
+# cluster LSF dispatching
+$SCRATCH/miniconda3/bin/snakemake -s vpipe.snake --use-conda --conda-prefix $SCRATCH/snake-envs -p --cluster 'bsub' --jobs 2
+
+# alternative for running everything from a single interactive SSH node
+bsub -I <<<"$SCRATCH/miniconda3/bin/snakemake -s vpipe.snake --use-conda --conda-prefix $SCRATCH/snake-envs -p --cores 2 --jobs 2"
```

Comment:
isn't --jobs the same as  --cores? Maybe unify throughout all examples.

And why ist: -bsub './vpipe ...'- enough?

Reply:
stray "cores" even if "jobs" is there: removed in PR #39 .
"cores" vs "jobs", currenlty snakemake doesn't make a distinction, but in theory "jobs" controls how many jobs are dispatched on the grid, whereas "cores" control the parallel CPU usage in the current machine. 

`bsus ./vpipe `

it _can_ work, but for very cryptic complex reasons it is strongly advised against in LSF manuals and lot of things can subtly go wrong (among other reasons: if the command is a script and isn't piped, bsub will not check the content of the script and react to job parameters written in the special fields)